packages/core/getBusyTimes.ts

```diff
@@ -359,11 +368,20 @@ export async function getBusyTimesForLimitChecks(params: {
     })}`
   );
 
+  let eventTypeIdFilter: Prisma.BookingWhereInput = { eventTypeId };
+  if (globalBookingLimits) {
+    if (bookingLimits && Object.keys(bookingLimits).length > 0) {
+      eventTypeIdFilter = { eventTypeId: { not: eventTypeId } };
```

Comment:
what's that filter for?

Reply:
This was added earlier as i had a different implementation in place, removed it now as implementation has changed.